src/env.rs

```diff
@@ -34,6 +34,7 @@ mod tests {
 
     #[test]
     fn hermit_dir_defaults_to_dot_config() {
+        env::remove_var("HERMIT_ROOT");
```

Comment:
This is actually might not be the best way to solve this.  It doesn't appear to interfere with the other environment variable test in this directory because of the sleep, but that's not a super ideal solution either.

Better solution: use a real concurrency tool to ensure that the code for both of these tests isn't run concurrently.